### ISSUE TITLE
exchangecalendar doesn´t take care about private tagged appointments

### ISSUE THREAD

**Issue by peddy1stg** [Ericsson/exchangecalendar#75]
private taggt appointments are schown.

**Comment by muthusuba** [comment]
@peddy1stg a little more information would be useful, please?

**Comment by peddy1stg** [comment]
Hi Muthu,

if you have two mailboxes with Delegation enabled between Mailbox A and
Mailbox B so Mailbox B is allowed to Access Information  from Mailbox A.
User of Mailbox A have an private appointment. User B Access the calender
of Mailbox A the private appointment will Show up.

If have investigated this and noticed that the EWS Protocoll  always
Transfer every Appoint. The Client have to filter the result and do´t Show
up the private one according the configured permission. (
https://grepular.com/Microsoft_Exchange_Private_Appointments_Arent_Private)

hope this helps to solve this Problem.

Kind regards

  Thomas


2015-01-16 16:23 GMT+01:00 Muthu Subramanian <notifications@github.com>:

**Comment by Elvon** [comment]
I would be very happy if you could implement this feature :-)

**Comment by bavincen** [comment]
Okay guys I'm little confuse.  We do delegate calendar?

**Comment by bavincen** [comment]
If yes. When delegating to other user you can uncheck the option for private items  in delegationcalendar..

**Comment by Elvon** [comment]
No, its about how EWS handles the sending of calendar information.
The Problem is that EWS sends all appointment information to the client.
Which means if you add a calendar of an other person you'll see all appointments, also the private ones incl. the subject and content.

You can test it if you have a Exchange Env.
Create a test user and then create an private appointment with Outlook or OWA.
After it add this calendar with your user account into lightning and you will see ;-) 

MS will not fix this issue .. they say:
Important: You should not rely on the Private feature to prevent other people from accessing the details of your appointments, contacts, or tasks. To make sure that other people cannot read the items that you mark as private, do not grant them read permission to your Calendar, Contacts, or Tasks folders. A person with read permission to access your folders could use programmatic methods or other e-mail applications to view the details of a private item. Use the private feature only when you share folders with people whom you trust.
https://support.office.com/en-za/article/Schedule-an-appointment-935cd8dd-4630-40d6-ad33-763c8ca4dc87#bm4

**Comment by muthusuba** [comment]
@Elvon is this fixed in the latest release, please?

**Comment by Elvon** [comment]
The issue is not fixed in v3.6-beta1. Checked it a minute ago.
